File: sqlite3/rhino/update/sqlite3_update.js
#! /usr/bin/java org.mozilla.javascript.tools.shell.Main
// ------------------------------------------------------------------
//
//	sqlite3_update.js
//
//						Oct/22/2014
//
// ------------------------------------------------------------------
importPackage(java.lang);
importPackage(java.sql);
load ("/var/www/data_base/common/rhino_common/sql_manipulate.js");
load ("/var/www/data_base/common/rhino_common/text_manipulate.js");

print	("*** 開始 ***");

var database = arguments[0];
var key_in=arguments[1];
var population_in=arguments[2];

print	(key_in,population_in);

Class.forName ("org.sqlite.JDBC");

var user = "";
var passwd = "";

var url = "jdbc:sqlite:" + database;

var conn = DriverManager.getConnection (url,user,passwd);

update_proc (conn,key_in,population_in);

conn.close();

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: sqlite3/jython/update/sqlite3_update.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	update/sqlite3_update.py
#
#					Jun/01/2011
#
# --------------------------------------------------------
import	sys
import	string
#
import java
from java.sql import DriverManager
# --------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc,update_proc
#
# --------------------------------------------------------
print ("*** 開始 ***")
java.lang.Class.forName("org.sqlite.JDBC")
#
sqlite3_file = sys.argv[1]
id_in = sys.argv[2]
population_in = string.atoi (sys.argv[3])
print ("%s\t%d" % (id_in, population_in))
#

url="jdbc:sqlite:" + sqlite3_file
user = ""
password = ""

conn = DriverManager.getConnection (url,user, password)
#
update_proc	(conn,id_in,population_in)
#
display_proc (conn)
#
conn.close ()
#
print ("*** 終了 ***")
#
# --------------------------------------------------------
